shared/chat/inbox/container/remote.js

```diff
@@ -48,11 +48,26 @@ const valuesCached = memoize3(
       .toArray()
 )
 
+// we only care to rerender if the inboxversion increases so
+// given the same inbox version give back the same metaMap
+const inboxVersionToMetaMap = memoize2(
+  (inboxVersion, metaMap) => metaMap,
+  (a, b) => {
+    // custom equality, if inboxVersion is the same, treat it as the same
+    if (typeof a === 'number') {
```

Comment:
this is lame but the next version of memoize-one will pass an idx so you can differentiate your params